### DISCUSSION TITLE
ffmpeg add logo  to stream

### DISCUSSION THREAD

**Discussion by shah1git** [bluenviron/mediamtx#2061] [Questions]
I have stream rtsp://localhost:8554/dub and i need the same stream with logo , additional  ffmpeg options -i logo.png -filter_complex "overlay=10:10" ,

My current config file looks like this
```
paths:

  dub:
    source: rtsp://myuser:mypass@dub321.dyndns-ip.com:554
```

what do I need to add to the config to get another stream with the logo?

**Comment by aler9** [comment]
Hello, this has been already answered here: https://github.com/bluenviron/mediamtx/discussions/1660#discussioncomment-5552039

[a comment by an automated account is not shown]
